src/components/Sidebar/components/NavItem/NavItem.spec.js

```diff
@@ -48,34 +48,20 @@ describe('NavItem', () => {
       expect(actual).toMatchSnapshot();
     });
 
-    it('should render children when selected', () => {
```

Comment:
Why did you remove these tests?

Reply:
`should render children when selected` was testing the Nested nav feature before we had the aggregator. It's outdated, but still passed as a false positive (doubled checked with @afonsodelgado)

`should render an icon` tests if an icon was rendered, and I removed it since this is now extensively tested in the `getIcon` util spec. However on review, it's true that the `getIcon` test suite checks which icon the util returns, but not its rendering. Adding the test back.